wordpress-mobile/WordPress-iOS
# Always update Widgets after logout/login

Fixes #15833

To test:

- install one or more widgets
- logout/login with different accounts and make sure that the widgets are updated

**Note**
When logging out and in several times with more than one widget installed, in some cases, I noticed a delay in updating some widgets of about 1 minute (sometimes less, sometimes not at all). I am not sure yet why this happened but would be good to know if you see the same (see video below)

<p align=center>
<img height=400 src=https://user-images.githubusercontent.com/34376330/107313623-8ec6ed80-6a58-11eb-99d3-fe98a13adf0a.gif>
</p>

cc @jkmassel 

PR submission checklist:
